### PR TITLE
Tw1

### PR DESCRIPTION
**Pull request by MenteeVelga51**
Please check and add Login form and Registration success page

### CODE REVIEW

File: src/components/Loginform/Loginform.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import './LoginForm.scss'
```
Comment (sliceem88) [inline src/components/Loginform/Loginform.jsx:1]: I don't see this LoginForm.scss file any where

Comment (sliceem88) [reply src/components/Loginform/Loginform.jsx:1]: Maybe you forgot to push it ??

File: src/components/RegistrationSuccessPage/RegistrationSuccessPage.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+    "use client";
+    import
```
Comment (sliceem88) [inline src/components/RegistrationSuccessPage/RegistrationSuccessPage.jsx:3]: You have key word `import` , but I don't see that you are importing something

File: src/components/RegistrationSuccessPage/RegistrationSuccessPage.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+    "use client";
+    import   
+    function Registrationsuccesspage () {
+      return <head>Registration success page</head>
+      function buildURL(item)
```
Comment (sliceem88) [inline src/components/RegistrationSuccessPage/RegistrationSuccessPage.jsx:6]: Syntaxis is wrong. Please see React documentation how it should look like, Or check your code from `LoginForm` component.

1. return should contain one element/tag
2. tag/element can have a lot of children
3. `function buildURL(item)` shoul be out of return statement
  You can check this example of component:
  
```
  import React from 'react';
export const renderButton = (props) => {
    const handleClick = () => {
        console.log('handleClick() props data from App: ' + props.dataFromApp)

    }
    return (
        <button onClick={handleClick}>Click</button>
    )
}
```

File: src/components/Loginform/Loginform.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+function LoginForm() {
+  return (
+  <div className="LoginForm">
+      <h2>Pierakstīšanās forma</h2>
+      <label for="username"><b>Lietotājvārds</b></label>
+      <input type="text" placeholder="Enter Username" id="username" required />
+      <label for="password"><b>Parole</b></label>
+      <input type="password" placeholder="Enter Password" id="password" required />
+      <button onclick="login()">Pierakstīties</button>
```
Comment (sliceem88) [inline src/components/Loginform/Loginform.jsx]: onclick event name is wrong, it should be `onClick`  - it is important
login() function itself should be before `return` statement

**Review by sliceem88 — CHANGES_REQUESTED** [review]

**Comment by sliceem88** [comment]
Also I don't see where your component is used

File: src/components/Loginform/Loginform.jsx (hunk 1 above)
Comment (MenteeVelga51) [reply src/components/Loginform/Loginform.jsx:1]: Will create the file

**Pull request closed without merging**